cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -171,26 +182,51 @@ func (sub EthereumSub) Start(completionEvent *sync.WaitGroup) {
 			completionEvent.Add(1)
 			go sub.Start(completionEvent)
 			return
+		case err := <-subHead.Err():
+			sub.Logger.Error(err.Error())
+			completionEvent.Add(1)
+			go sub.Start(completionEvent)
+			return
+		case newHead := <-heads:
```

Comment:
I am getting some issues running this locally. The tx's are going through instantly when I do the following:

1. run `./test/integration/start-integration-env.sh` from the root directory

2. Query the account value and take note of the ceth balance.

3. Now, burn some more eth on the ganache chain. This results in the sifnode account being given ceth instantly without the 30 block wait.

Happy to walk you through what I'm doing on my local machine so that we can debug.

Reply:
When I checked out to commit `b826e670a5381e3d1c70a32b54da541e712a5553`, which was before I merged the code with develop, everything worked fine on your branch except for transfers that were over or equal to 1e19 as that code wasn't on your branch before you started.